contracts/bonding/IBondingManager.sol

```diff
@@ -16,6 +16,7 @@ contract IBondingManager {
     event Rebond(address indexed delegate, address indexed delegator, uint256 unbondingLockId, uint256 amount);
     event WithdrawStake(address indexed delegator, uint256 unbondingLockId, uint256 amount, uint256 withdrawRound);
     event WithdrawFees(address indexed delegator);
+    event EarningsClaimed(address indexed delegator, address indexed delegate, uint256 rewards, uint256 fees, uint256 startRound, uint256 endRound);
```

Comment:
Since the other events that include both a delegator and a delegate address specify the delegator address and then the delegate address what do you think about doing the same in this event?

Reply:
Oh yeah sounds good, I didn't notice

Reply:
fixed in d8c8a5f